src/custom/components/Header/index.tsx

```diff
@@ -40,5 +61,5 @@ export const LogoImage = styled.img.attrs(props => ({
 `
 
 export default function Header() {
-  return <HeaderModWrapper statusLabel={<AppStatusWrapper>{appStatus}</AppStatusWrapper>} />
```

Comment:
Instead of passing `appStatus` as a Prop, we can keep it in this file, by using it as `content` for the CSS `::after`. On hover of the logo, the ALPHA label also rotates along now.